student.rb

```diff
@@ -1,14 +1,22 @@
-require './person'
+require_relative 'person'
+require_relative 'classroom'
+require_relative 'rental'
 
 class Student < Person
-  attr_accessor :classroom
+  attr_reader :classroom, :rentals
```

Comment:
- [ ] Please make sure that a person has many rentals. Instead of doing so in the `teacher` or `student`, you can do that in `person`. 

```ruby
class Person
  ...
  ...
  def add_rental(book, date)
    Rental.new(date, book, self)
  end
  ...
  ...
end
```